classes/uabb-global-functions.php

```diff
@@ -1,88 +1,77 @@
-<?php 
-
+<?php
 /**
- * = Global functions
- *
- * array_replace_recursive()
```

Comment:
why did you remove all theme function comments?

Reply:
Added